spec/abstractops.html

```diff
@@ -1660,6 +1660,7 @@ <h1>
         _fields_: an Object,
         _fieldNames_: a List of property names,
         _requiredFields_: ~partial~ or a List of property names,
+        optional _extraFieldDescriptors_: a List of Calendar Field Descriptor Records
```

Comment:
#2472 also fixes #2465, and I hope to have that one merged before this normative change comes up for presentation in plenary. Is this fix incompatible with that PR or can it be rebased on top of that PR and/or folded into another commit in this PR?

Reply:
Handled by rebase; it's still a "Normative: Remove the ECMA-402 PrepareTemporalFields override" commit in this PR.